Sources/Purchasing/Configuration.swift

```diff
@@ -207,19 +212,32 @@ import Foundation
 
 // MARK: - Public Keys
 
-internal extension Configuration {
+public extension Configuration {
 
-    /// Defines how strict ``EntitlementInfo`` verification ought to be.
-    @objc(RCEntitlementVerificationLevel)
-    enum EntitlementVerificationLevel: Int {
+    // TODO: link to docs
```

Comment:
Guessing this will be done after this is added to the docs right? We should just make sure we don't forget.

Reply:
I just filed a Jira for this 👍🏻